services/graph/pkg/identity/ldap.go

```diff
@@ -20,10 +20,29 @@ import (
 )
 
 const (
-	givenNameAttribute = "givenname"
-	surNameAttribute   = "sn"
+	_givenNameAttribute        = "givenname"
+	_surNameAttribute          = "sn"
+	_ldapGroupOfNamesAttribute = "(objectClass=groupOfNames)"
+	_ldapGroupMemberAttribute  = "member"
```

Comment:
Aren't these all defined in the user attribute map?

Reply:
Hm, good catch. Actually they aren't. `givenName` and `surName` should probably move there. 

`ldapGroupOfNamesAttribute = "(objectClass=groupOfNames)"` seems to be a bit of a misnomer. It's a filter not an attribute. But I don't think we even need that. I think we can do the disabled group lookups with the default `(objectclass=*)` filter as we're doing a direct base level DN lookup.

I think the `_ldapGroupMemberAttribute` can stay here as a constant. We don't need it to be configurable for the disable by group feature.

Reply:
BTW, I am fine to get this addressed in a follow up PR. As this needs adjustments in a couple of more places (I think).